### PR TITLE
jetbrains webstorm 2019.1

### PR DESCRIPTION
**Pull request by x80486**

### CODE REVIEW

File: com.jetbrains.WebStorm.yaml [hunk 1]
```diff
@@ -24,21 +25,15 @@ modules:
...
 
-  - name: WebStorm
+  - name: webstorm
     buildsystem: simple
     build-commands:
       - unzip webstorm_logos.zip
       - install -D apply_extra /app/bin/apply_extra
-      - install -Dm755 webstorm-desktop.sh "${FLATPAK_DEST}/bin/webstorm-desktop"
-      - install -Dm644 logo.svg "${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg"
-      - icon_in=logo.svg;
```
Comment (x80486) [inline com.jetbrains.WebStorm.yaml]: @FakeShemp, I removed the whole `rsvg-convert` Jiu Jitsu. I don't think it's needed, but I may be wrong. I see the correct icons when I provide the `SVG`. Let me know in any case :sunglasses:

Comment (FakeShemp) [reply com.jetbrains.WebStorm.yaml]: I'm not sure all distros handle svg-only icons. I would prefer to only use them, but others seem to object. Maybe someone knowledgeable can give some insight.

Comment (x80486) [reply com.jetbrains.WebStorm.yaml]: What I meant was that, somehow, Flathub builds generate the appropriate `PNG` files/icons (in the correct locations) when you provide an `SVG`; take a look at the build's log. Now, it doesn't generate all the resolutions :innocent:

**Comment by FakeShemp** [comment]
Works nice. Great job :+1:

**Pull request merged**